src/model/IPoll.ts

```diff
@@ -108,7 +108,7 @@ export interface IPoll {
 const pollSchema = Joi.object<IPoll, true>({
   title: Joi.string().min(1).required(),
   question: Joi.string().min(1).required(),
-  description: Joi.string().min(1).required(),
+  description: Joi.string().min(0).required(),
```

Comment:
why did we change that here?

Reply:
Because is it currently not possible to create polls with an empty description (see changeset)